src/Test/Unit/Config/Validator/Deploy/DatabaseSplitConnectionTest.php

```diff
@@ -78,14 +79,19 @@ public function testMessageValidate()
             ->willReturn($dbConfiguration);
         $this->resultFactoryMock->expects($this->once())
             ->method('error')
-            ->with('Split database configuration was detected in the property DATABASE_CONFIGURATION of the'
-                . ' file .magento.env.yaml:' . PHP_EOL
-                . '- connection: checkout' . PHP_EOL
-                . '- connection: sales' . PHP_EOL
-                . '- slave_connection: checkout' . PHP_EOL
-                . '- slave_connection: sales' . PHP_EOL
-                . 'Magento Cloud does not support a custom split database configuration,'
-                . ' such configurations will be ignored');
+            ->with(
+                'Detected split database configuration in the DATABASE_CONFIGURATION property of the'
+                    . ' file .magento.env.yaml:' . PHP_EOL
+                    . '- connection: checkout' . PHP_EOL
+                    . '- connection: sales' . PHP_EOL
+                    . '- slave_connection: checkout' . PHP_EOL
+                    . '- slave_connection: sales' . PHP_EOL
+                    . 'Magento Cloud does not support a custom split database configuration,'
```

Comment:
Consider the following suggestion.

```suggestion
                    . 'Magento Cloud does not support custom connections in the split database configuration,'
```